### ISSUE TITLE
[Adaptive Cards- Expenses Report]: While navigating on the 'Expand' button, screen reader is not conveying descriptive information.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9023]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Turn on screen reader using caps + spacebar key.

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Expenses Report' link and press enter key to activate it.
4. Press tab key and navigate to expand button and observe the issue.

**Actual Result:**
While navigating on the expand button present on the expenses report screen reader is not conveying descriptive information as category is not associated with the button. It is announcing only as **'Expand button.'**
Also, after activating the expand button, screen reader is not announcing the changed state as it is only announcing as expand button.

**Expected Result:**
Screen reader should announce the descriptive information while navigating on the expand button. It should announce as **'Expand Air Travel expenses button.'**
Also, when user activates expand button it should announce as expanded.

**Note:**
Same issue is repro with NVDA as well.

**User Impact:**
It will impact the screen reader user as they will not be able to know descriptive information about the expand button also there are 3 expand buttons are present so user will get confused if the screen reader will announce the same information for the all buttons,

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![While navigating on the 'Expand' button, screen reader is not conveying descriptive information](https://github.com/user-attachments/assets/0269626e-4f0d-4389-88f0-538a7324a14c)

https://github.com/user-attachments/assets/3cf72bb9-ca7c-4361-b16e-4f13f8c8f921

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro on the updated UI, hence closing this issue.

https://github.com/user-attachments/assets/ea3eb2a1-942b-4c9c-bf1b-9bc1600442e7
